packages/user/src/User/User.purs

```diff
@@ -157,16 +160,19 @@ getUser maybeInvalidateCache uuid token = do
       Console.info "User fetched successfully"
       pure user
 
+isAdminUser :: Effect Boolean
+isAdminUser = isJust <$> LocalStorage.getItem "adminUuid"
```

Comment:
Should this be `isAdmin` ?

Reply:
Yes it should, that was left over from the previous version. Will fix.